blog/en/docusaurus.config.js

```diff
@@ -105,6 +105,19 @@ module.exports = {
       trackingID: 'GTM-K24PRPS',
     },
   },
+  scripts: [
+    {
+      src: 'https://widget.kapa.ai/kapa-widget.bundle.js',
+      'data-website-id': '23b59d9a-682e-4c3d-9e83-bf2ee85cdc19',
+      'data-project-name': 'APISIX',
+      'data-project-color': '#E8442E',
+      'data-project-logo':
+        'https://user-images.githubusercontent.com/52855735/62005987-53faae80-b16d-11e9-96cc-9dba3470f5b2.png',
+      'data-modal-disclaimer': 'This is a custom LLM for APISIX with access to all developer documentation, GitHub issues and discussions.',
+      'data-modal-example-questions': 'Does APISIX have a UI?,How do I use REGEX for uri routes?',
```

Comment:
can we set more questions?
- What is Apache APISIX?
- What protocols can Apache APISIX handle?
- How to proxy WebSocket connection?
- How does Apache APISIX work with Keycloak?

Reply:
OK. I will add more questions.

Reply:
I think we can follow the user journey.
> What is APISIX -> How to deploy APISIX -> How to use APISIX

like what skywalking does. refer: https://github.com/apache/skywalking-website/blob/fe54f2541fcabba6ee49c6497ca9daaed2a91795/themes/docsy/layouts/partials/scripts.html#L34

![image](https://github.com/apache/apisix-website/assets/71714656/f0393381-61bb-4c4f-b02c-26bfc8eec906)


What do you think? @moonming

Reply:
This repo can be divided into three parts: `website`, `doc` and `blog(en or zh)`. Each parts contains different documents, do we need different questions for different parts?

Reply:
> I think we can follow the user journey.
> 
> > What is APISIX -> How to deploy APISIX -> How to use APISIX
> 
> 
> 
> like what skywalking does. refer: https://github.com/apache/skywalking-website/blob/fe54f2541fcabba6ee49c6497ca9daaed2a91795/themes/docsy/layouts/partials/scripts.html#L34
> 
> 
> 
> ![image](https://github.com/apache/apisix-website/assets/71714656/f0393381-61bb-4c4f-b02c-26bfc8eec906)
> 
> 
> 
> 
> 
> What do you think? @moonming 

Good answers come from the right questions. I don't think general questions are valuable

Reply:
> > I think we can follow the user journey.
> > > What is APISIX -> How to deploy APISIX -> How to use APISIX
> > 
> > 
> > like what skywalking does. refer: https://github.com/apache/skywalking-website/blob/fe54f2541fcabba6ee49c6497ca9daaed2a91795/themes/docsy/layouts/partials/scripts.html#L34
> > ![image](https://private-user-images.githubusercontent.com/71714656/295405561-f0393381-61bb-4c4f-b02c-26bfc8eec906.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ4NjIzMDcsIm5iZiI6MTcwNDg2MjAwNywicGF0aCI6Ii83MTcxNDY1Ni8yOTU0MDU1NjEtZjAzOTMzODEtNjFiYi00YzRmLWIwMmMtMjZiZmM4ZWVjOTA2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTEwVDA0NDY0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM2OGFmYThmMWM1MjZmMjRmZTYxZGVlNWE2NjQ1YmVkYmRmOTFkNjQ3NTI2ZWE1MGZjNGYzMjU4ODBmYzVkY2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.n8x60ohlBTh4IE4k3X2Axg08ozEPXYI-xzDghTpwr38)
> > What do you think? @moonming
> 
> Good answers come from the right questions. I don't think general questions are valuable

Very good point. 

But what I mean here is that we can set up the problem from these aspects. Didn't mean to set the specific question up as `How to use APISIX`, `How to deploy APISIX`, etc. We need to set up some valuable questions from the user's perspective to help users use APISIX more effectively, right?

Reply:
OK, got it. You can make a try😀